apps/docs/src/.vitepress/components/ComponentRoadmap.vue

```diff
@@ -61,6 +61,10 @@ const shouldShowAllButton = computed(() => {
       regularly, we will keep it up to date with our progress.
     </p>
 
+    <p class="components__description">
+      <b>Tip:</b> You can click on a component to see its implementation and documentation.
```

Comment:
should we specifiy that this only works on implemented/started components? or is this common sense